src/test/java/Homework17.java

```diff
@@ -0,0 +1,27 @@
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+import java.time.Duration;
+public class Homework17 extends BaseTest {
+
+    @Test
+    public static void addSongToPlaylist() throws InterruptedException {
+        String newSongAddNotificationMessage = "Added 1 song into";
+        loginUrl();
+        enterEmail("catsaremyfavorite@gmail.com");
+        enterPassword("te$t$tudent");
+        clickSubmit();
+        findSong("Take my Hand");
+        clickViewAllButton();
+        selectFirstSong();
+        clickAddToButton();
+        selectPlaylist();
+      Assert.assertTrue(getNotificationMessage().contains(newSongAddNotificationMessage));
+    }
+}
+
+
```

Comment:
Please, add Homework16 in TestNG.xml to run tests

For example

```
<?xml version="1.0" encoding="UTF-8"?>
<!DOCTYPE suite SYSTEM "http://testng.org/testng-1.0.dtd" >
<suite name="TestNG">
    <test name="Example Test" preserve-order="false">
        <classes>
            <class name="Homework17"/>
        </classes>
    </test>
</suite>
```

Reply:
I don't understand. I had Homework16 in the TestNG folder, but it disappeared once I ran my code through commit and   pushed it.